media.install

```diff
@@ -62,4 +63,13 @@ function media_install() {
   $source = drupal_get_path('module', 'media') . '/images/icons';
   $destination = \Drupal::config('media_entity.settings')->get('icon_base');
   media_entity_copy_icons($source, $destination);
+  // Add an icon for Media if we aren't installing from configuration.
+  if (!\Drupal::isConfigSyncing()) {
+    $data = file_get_contents(dirname(__FILE__) . '/images/media_embed_icon.png');
+    $file = file_save_data($data, 'public://media_embed_icon.png', FILE_EXISTS_REPLACE);
+    // Set file uuid same as default config.
+    $uuid = (new ExtensionInstallStorage(\Drupal::service('config.storage')))->read('embed.button.media')['icon_uuid'];
```

Comment:
This is very strange. We should rather update the embed button with the UUID of the new file.